samples/sdl/main.c

```diff
@@ -187,7 +186,7 @@ static void
 quit(int rc)
```

Comment:
nit: I don't consider `quit` to be a good name (the name sounds a bit like it's standard library functionality to me). Can't think of a better name right now, though.